src/pages/home/ReportScreen.js

```diff
@@ -170,20 +173,13 @@ class ReportScreen extends React.Component {
         return !getReportID(this.props.route) || isLoadingInitialReportActions || !this.props.report.reportID;
     }
 
-    /**
-     * Persists the currently viewed report id
-     */
-    storeCurrentlyViewedReport() {
+    fetchReportIfNeeded() {
```

Comment:
Renamed as we are not storing the reportID anymore (didn't really like that we are resetting the hide composer thing in the same method anyway).

Reply:
NAB: to me, the name should be `fetchReport` because the `IfNeeded` part is an implementation detail of the method which the outside world doesn't need to know about.